libethereum/State.h

```diff
@@ -326,29 +395,45 @@ class State
 
 	ChangeLog const& changeLog() const { return m_changeLog; }
 
-private:
+   private:
 	/// Turns all "touched" empty accounts into non-alive accounts.
 	void removeEmptyAccounts();
 
-	/// @returns the account at the given address or a null pointer if it does not exist.
-	/// The pointer is valid until the next access to the state or account.
+	/// @returns the account at the given address or a null pointer if it does
+	/// not exist. The pointer is valid until the next access to the state or
+	/// account.
 	Account const* account(Address const& _addr) const;
 
-	/// @returns the account at the given address or a null pointer if it does not exist.
-	/// The pointer is valid until the next access to the state or account.
+	/// @returns the account at the given address or a null pointer if it does
+	/// not exist. The pointer is valid until the next access to the state or
+	/// account.
 	Account* account(Address const& _addr);
 
 	/// Purges non-modified entries in m_cache if it grows too large.
 	void clearCacheIfTooLarge() const;
 
 	void createAccount(Address const& _address, Account const&& _account);
 
-	OverlayDB m_db;								///< Our overlay for the state tree.
-	SecureTrieDB<Address, OverlayDB> m_state;	///< Our state tree, as an OverlayDB DB.
-	mutable std::unordered_map<Address, Account> m_cache;	///< Our address cache. This stores the states of each address that has (or at least might have) been changed.
-	mutable std::vector<Address> m_unchangedCacheEntries;	///< Tracks entries in m_cache that can potentially be purged if it grows too large.
-	mutable std::set<Address> m_nonExistingAccountsCache;	///< Tracks addresses that are known to not exist.
-	AddressHash m_touched;						///< Tracks all addresses touched so far.
+	OverlayDB m_db;  ///< Our overlay for the state tree.
+	SecureTrieDB<Address, OverlayDB>
+		m_state;  ///< Our state tree, as an OverlayDB DB.
+	mutable std::unordered_map<Address, Account> m_cache;  ///< Our address
+														   ///< cache. This
+														   ///< stores the
```

Comment:
This is caused by "reflow comments" and max column 80. We can turn it off for now, but we can also fix the comments by moving them before the variables instead of having them after with `///<`.

Reply:
Also fixed.